src/EnergyPlus/WeatherManager.cc

```diff
@@ -3490,6 +3511,9 @@ namespace WeatherManager {
                     TomorrowWindSpeed(CurTimeStep, Hour) = WindSpeed;
                     TomorrowWindDir(CurTimeStep, Hour) = WindDir;
                     TomorrowLiquidPrecip(CurTimeStep, Hour) = LiquidPrecip;
+                    TomorrowTotalSkyCover(CurTimeStep, Hour) = TotalSkyCover;
+                    TomorrowOpaqueSkyCover(CurTimeStep, Hour) = OpaqueSkyCover;
+                    HrIRHoriz_Weather(Hour) = IRHoriz;
```

Comment:
@mjwitte Agree

Reply:
An update: due to the `if` condition statement on Line 3602 (`NumOfTimeStepInHour > 1', Lines 3516-3535 should still be kept. Otherwise for the situation of using hourly timestep (when NumOfTimeStepInHour ==1), the HorizIR and SkyTemp would not be filled and updated correctly since this whole block of interpolation from Lines 3602-3745 would not run when `NumOfTimeStepInHour == 1`.